azurerm/internal/services/applicationinsights/application_insights_data_source.go

```diff
@@ -35,6 +35,11 @@ func dataSourceArmApplicationInsights() *schema.Resource {
 				Computed: true,
 			},
 
+			"connection_string": {
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
We should mark this as sensitive here as well

Reply:
Interesting.. I was copying the instrumentation_key property which is marked sensitive at the resource and _not_ sensitive at the data source level. Did you want to be consistent with instrumentation_key or mark connection_string as sensitive?

Reply:
It's weird that it's marked as sensitive in one place and not the other with `instrumentation_key`. I'm not sure why that's the case but let's keep it consistent with the resource and mark `connection_string` as Sensitive.

Reply:
Sounds good, I was guessing it was an oversight at some point. I'll get those changes pushed up.